### PR TITLE
Fix x describe do not work with weighted routes to different hosts

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
Fix https://github.com/istio/istio/issues/43368

### CODE REVIEW

File: releasenotes/notes/43398.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+releaseNotes:
```
Comment (GregHanson) [inline releasenotes/notes/43398.yaml:4]: can you link the issue here?

Comment (hanxiaop) [reply releasenotes/notes/43398.yaml:6]: Done

**Review by GregHanson — APPROVED** [review]

File: releasenotes/notes/43398.yaml (hunk 1 above)
Comment (GregHanson) [reply releasenotes/notes/43398.yaml:6]: thanks!

**Comment by hanxiaop** [comment]
@istio/wg-user-experience-maintainers PTAL, thanks.

File: istioctl/cmd/describe.go [hunk 2]
```diff
@@ -1041,36 +1053,38 @@ the configuration objects that affect that service.`,
 
 			writer := cmd.OutOrStdout()
 
-			pods, err := client.CoreV1().Pods(ns).List(context.TODO(), metav1.ListOptions{})
-			if err != nil {
-				return err
+			labels := ""
+			for k, v := range svc.Spec.Selector {
+				labels += fmt.Sprintf("%s=%s,", k, v)
```
Comment (zhlsunshine) [inline istioctl/cmd/describe.go:1058]: how about to define labels as a string array, then use `strings.Join()` to complete such splice? then it's unnecessary for you to trim the separator in tail in L#1060 ?

Comment (hanxiaop) [reply istioctl/cmd/describe.go]: @zhlsunshine Thanks, fixed.

File: releasenotes/notes/43398.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+issue:
+  - 43368
+releaseNotes:
+  - |
+    **Fixed** `istioctl experimental describe` doesn't show all weighted routes when the VirtualService is defined to split traffic across multiple services.
```
Comment (dhawton) [inline releasenotes/notes/43398.yaml:8]:
```suggestion
    **Fixed** `istioctl experimental describe` not showing all weighted routes when the VirtualService is defined to split traffic across multiple services.
```

**Review by zhlsunshine — APPROVED** [review]
LGTM

**Pull request merged**